react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
Sure, it is included, but I don't see where we actually use the functionality, I removed it and the App worked as before?